c++/src/capnp/membrane.c++

```diff
@@ -234,23 +234,23 @@ public:
 
   AnyPointer::Pipeline sendForPipeline() override {
     return AnyPointer::Pipeline(kj::refcounted<MembranePipelineHook>(
-        PipelineHook::from(inner->sendForPipeline()), policy->addRef(), reverse));
+        PipelineHook::from(inner->sendForPipeline()), policy.addRef(), reverse));
   }
 
 private:
   kj::Own<RequestHook> inner;
-  kj::Own<MembranePolicy> policy;
+  kj::Rc<MembranePolicy> policy;
```

Comment:
ugh... It bugs me that we increment the refcount twice when MembraneRequestHook is created and decrement it twice on destruction, once here and once for the `capTable` member below.